contrib/ansible/openshift/register.sh

```diff
@@ -33,7 +33,9 @@ while read -u 9 line;do
   remote_name=$(echo $line | awk -F' ' '{print $1}')
   group_name=$(echo $line | awk -F' ' '{print $2}')
   index_value=$(echo $line | awk -F' ' '{print $3}')
+  index_value=$(( index_value + 1 )) 
```

Comment:
Why do we increment by one here?  Are we converting from a zero based scheme to a one based scheme?